### PR TITLE
Add LPD_YuNet for License Plate Detection

### PR DESCRIPTION
**Pull request by fengyuentau**
Checklist:
- [x] put model
- [x] impl (lpd_yunet.py)
- [x] demo
- [x] put license
- [x] license plate images for quantization and benchmark
- [x] quantize (managed to quantize with inc)
- [x] benchmark impl
- [x] collect benchmark results

### CODE REVIEW

**Comment by ShiqiYu** [comment]
The author among with the copyright holder should be added.

**Comment by fengyuentau** [comment]
Benchmark results of `INTEL-CPU` will be re-collected on our new dev machine (CPU: i7 12700K). So it is not provided for this model.

@ShiqiYu @kaingwade @zihaomu Please take a look.

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Guidelines:
 |-------|------------|----------------|--------------|-----------------|--------------|-------------|
 | [YuNet](./models/face_detection_yunet)                | 160x120  | 1.45   | 6.22    | 12.18  | 4.04   | 86.69 |
 | [SFace](./models/face_recognition_sface)              | 112x112  | 8.65   | 99.20   | 24.88  | 46.25  | ---   |
+| [LPD-YuNet](./models/license_plate_detection_yunet/)  | 320x240  | ---    | 168.03  | 56.12  | 154.20\* |       |
```
Comment (kaingwade) [inline README.md:21]: Channel wise quantized model runs slow on NPU. Is it possible to add the speed of tensor wise quantized model?

Comment (fengyuentau) [reply README.md:21]: unfortunately neural compressor does not provide any method to control that for now (see this issue https://github.com/intel/neural-compressor/issues/82). I will try [openppl-ppq](https://github.com/openppl-public/ppq) in the next patch.

**Review by zihaomu — APPROVED** [review]
In my opinion, it's okay to include the result of `Per-channel quantized` model at the moment.

**Comment by fengyuentau** [comment]
Yes, I agree. But we need to find a way to quantize model in per-tensor mode if onnxruntime quantization tool cannot work for all models. So merge as is and will update once the solution is found.

**Comment by zihaomu** [comment]
[Intel® Neural Compressor v1.12 Release](https://github.com/intel/neural-compressor/releases/tag/v1.12) has supported the per-tensor quantization (6 days ago).

[related comment](https://github.com/intel/neural-compressor/issues/82#issuecomment-1144315404)

**Pull request merged**